coralproject/talk
# [CORL-470] PAT Fix

## What does this PR do?

Fixed bug with token generation and verification.

## How do I test this PR?

1. Create a PAT via the `createToken` mutation.
2. Use the `signedToken` to perform a request against the API, notice the lack of error as a result of using the PAT.
